docs/F-Refinements/string-library.md

```diff
@@ -36,7 +44,7 @@ The header file that contains the prototypes for these library functions is:
 
 ## String Length
```

Comment:
```suggestion
### String Length
```

I think the headings for String Length, Copy, Compare, and Concatenate could be third level headings since they are all under String Functions. This would make the sidebar more clear for users such as:

![image](https://user-images.githubusercontent.com/59708190/141693624-1e2681ea-1f3a-4ed6-aa8b-d21035e3d5ff.png)

Reply:
That's right. Thanks for your suggestion.